src/redux/hub/hub.types.ts

```diff
@@ -22,10 +24,27 @@ export type HubImage = {
   avatar: string | null;
 };
 
+export type FilterMap = {
+  name: string;
+  selected: boolean;
+  count: number;
+};
+export type Filter = {
+  filterLabel: string;
+  values: FilterMap[];
+};
+
+export type FilterParams = {
+  kind: string[];
+  keywords: string[];
+  name?: string | null;
+};
+
 export type HubState = {
   images: HubImage[];
   loading: boolean;
   error: Error | null;
+  selectedFilters: string[]
```

Comment:
Since we have a finite amount of Filters, we should put them into their own type? Like type FilterLabel = Encoder | Indexer | ...